code-samples/serving/websockets-go/config/service.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: websockets-go
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+      - image: ko://github.com/knative/docs/code-samples/serving/websockets-go/cmd/server
```

Comment:
The other samples have docker.io/{username}/grpc-ping-go, so let's keep consitency.

Reply:
In the past we added [buildx](https://github.com/knative/docs/pull/5273) and since it touches many files any other addition should be more visible and get proper reviews. If something does not work we can open another issue and fix it (migrate to ko etc).

Reply:
@rhuss @pierDipi wdyth? Any objections to stamp?